plugins/BEdita/API/tests/TestCase/Utility/JsonApiTest.php

```diff
@@ -536,11 +642,40 @@ public function testFallbackLinks()
                     ],
                 ],
             ],
+            '_schema' => [
+                'documents' => [
+                    '$id' => '/model/schema/documents',
+                    'revision' => TestConstants::SCHEMA_REVISIONS['documents'],
+                ]
+            ]
         ];
 
         $result = JsonApi::formatData(TableRegistry::get('Documents')->get(2));
         $result = json_decode(json_encode($result), true);
 
         static::assertEquals($expected, $result);
     }
+
+    /**
+     * Test `schemaInfo` method
+     *
+     * @return void
+     * @covers ::schemaInfo
+     * @covers ::resetSchemaInfo
+     */
```

Comment:
add `@covers ::schemaRevision`? maybe not necessary: method is already covered by `testSchemaRevision`

Reply:
you mean `JsonSchema::schemaRevision` ? 
it's already covered in its own test case class, don't needed here